api/krusty/extendedpatch_test.go

```diff
@@ -1213,3 +1066,44 @@ spec:
     app: busybox
 `)
 }
+
+func TestTargetMissingPatchJson6902Error(t *testing.T) {
```

Comment:
We don't seem to have tests for
* `allowNoTargetMatch`
* json patches under the `patches` field